data/core/editor/items.cfg

```diff
@@ -408,6 +585,42 @@
         name= _ "Staff"
     [/item]
 
+    [item]
+        id=staff2
+        image=items/staff-2.png
+        name= _ "Staff"
+    [/item]
+
+    [item]
+        id=staff-druid
+        image=items/staff-druid.png
+        name= _ "Druid Staff"
+    [/item]
+
+    [item]
+        id=staff-magic
+        image=items/staff-magic.png
+        name= _ "Magic Staff"
+    [/item]
+
+    [item]
+        id=staff-magic2
+        image=items/staff-magic-2.png
+        name= _ "Magic Staff"
+    [/item]
+
+    [item]
+        id=staff-magic-wand
+        image=items/staff-magic-wand.png
+        name= _ "Magic Wand"
+    [/item]
+
+    [item]
+        id=staff-plain
+        image=items/staff-plain.png
+        name= _ "Plain Staff"
```

Comment:
I wonder if maybe this should be changed to "quarterstaff"? Looking at it, I _think_ that's what it's meant to be, right? I'm not 100% sure though…

Reply:
I recognise there may be a difference between the name that players see and the file naming for the sake of grouping similar items together. I wouldn't know what the original intent of each image is, though. Maybe @doofus-01 can comment on what the intention was? I note from https://opengameart.org/content/wesnoth-items that it's just labelled as a 'plain, unmagical staff', not necessarily an implication that it's meant for physical martial arts, though.

Reply:
I'd say it doesn't matter.  These are just images for content creators, they can use them in whatever way makes sense.

Reply:
I agree with doofus-01. The id= and name= fields do not matter here since after placement the image is WML-coded as 
```ini
[item]
    x=21
    y=21
    image="items/staff-plain.png"
[/item]
```

Reply:
Thanks for confirming. So the important thing is the path name.